pkg/goDB/DBWorkManager.go

```diff
@@ -67,14 +67,15 @@ type DBWorkManager struct {
 }
 
 // NewDBWorkManager sets up a new work manager for executing queries
-func NewDBWorkManager(dbpath string, iface string, numProcessingUnits int) (*DBWorkManager, error) {
+func NewDBWorkManager(query *Query, dbpath string, iface string, numProcessingUnits int) (*DBWorkManager, error) {
```

Comment:
This is just a trivial clean-up. I realized that w were handing over tons of copies of the original query all the way down to processing (with each workload), although it is a static piece of information that can be made available as struct variable in the `DBWorkManager`.